recipes/txaio/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "txaio" %}
+{% set version = "2.5.2" %}
+{% set sha256 = "321d441b336447b72dbe81a4d73470414454baf0543ec701fcfecbf4dcbda0fe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
```

Comment:
For this particular case, I'm going to go with Option 1. This package is a prerequisite for developing a recipe for autobahn python. It has similar set of optional dependencies and really depends on what backend (asyncio or twisted) the user wants to use. I'll just follow the style of the basic pip install that doesn't automatically install any particular backend and leaves it to the user.